packages/amplify_auth_plugin_interface/lib/src/types/AuthNextStep.dart

```diff
@@ -17,9 +17,9 @@ import 'package:flutter/foundation.dart';
 import 'AuthCodeDeliveryDetails.dart';
 
 class AuthNextStep {
-  Map<dynamic, dynamic> additionalInfo;
-  AuthCodeDeliveryDetails codeDeliveryDetails;
-  AuthNextStep({@required codeDeliveryDetails, additionalInfo = const {}}) {
+  late Map<dynamic, dynamic> additionalInfo;
+  late AuthCodeDeliveryDetails codeDeliveryDetails;
```

Comment:
Some context on why late is required here will be helpful.

Reply:
Do you think it's necessary to add comments to the dart file to explain?  I'll write the explanation here for now.  If a type is non nullable, then the compiler is checking to see that it is initialized somewhere in the constructor or variable declaration, if it can't find that it will throw an error until late keyword is added. 

1) Even though we have a default value of const {} for additionalInfo, that does not satisfy the compiler.  If you remove the late modifier, the compiler will complain that the field isn't initialized.  

2) codeDeliveryDetails is only initialized within the constructor method and not within the constructor parameters or when it was declared.